src/connection/connection.rs

```diff
@@ -2167,7 +2184,7 @@ impl Connection {
         let out = &mut out[st.written..];
         if (pkt_type != PacketType::OneRTT && pkt_type != PacketType::ZeroRTT)
             || self.is_closing()
-            || out.len() <= frame::MIN_STREAM_OVERHEAD
+            || out.len() <= frame::MAX_STREAM_OVERHEAD
```

Comment:
Can you explain something more on the starvation problems and why this changes make sense in comments? (if I remember correct it is the RR issue?)

Reply:
Also, if the bug still holds, report it to the official repo.

Reply:
The bug has been fixed in the official repo. The modifications made here are consistent with the official version.

Reply:
In that way, please merge the official repo into our repo and do not modify them by ourselves.